src/components/ContactCard/ContactForm/ContactFormField.jsx

```diff
@@ -15,7 +15,7 @@ class ContactFormField extends React.Component {
   }
 
   render() {
-    const { name, icon, label, isArray, renderInput, t } = this.props
+    const { name, icon, isArray, renderInput, t } = this.props
```

Comment:
Is the label prop still used in this component at all? Otherwise we should remove it from the proptypes as well.

Reply:
oops bien vu